core/src/main/java/nav/statusplattform/core/openingHours/ServicesUpTimeRenumerator.java

```diff
@@ -0,0 +1,248 @@
+package nav.statusplattform.core.openingHours;
+
+import nav.statusplattform.core.entities.RecordEntity;
+import nav.statusplattform.core.enums.ServiceStatus;
+import nav.statusplattform.core.repositories.RecordRepository;
+import org.fluentjdbc.DbContext;
+
+
+import java.time.*;
+import java.time.temporal.ChronoUnit;
+import java.util.*;
+
+import static java.time.LocalDate.*;
+
+public class ServicesUpTimeRenumerator {
+    RecordRepository recordRepository;
+    static UpTimeTotal uptimeTotal;
+
+    public ServicesUpTimeRenumerator(DbContext context) {
+        this.recordRepository = new RecordRepository(context);
+    }
+
+    public UpTimeTotal calculateUpTimeForService(UUID serviceId, LocalDate DateEntryFrom, LocalDate DateEntryTo, String rule) {
```

Comment:
```suggestion
    public UpTimeTotal calculateUpTimeForService(UUID serviceId, LocalDate dateEntryFrom, LocalDate dateEntryTo, String rule) {
```
Lowercase variables, PascalCase classes.